README.md

```diff
@@ -16,16 +15,173 @@ To set up the project, download these tools:
 
 These Go and Dep versions are compliant with the `buildpack` used by Prow. For more details read [this](https://github.com/kyma-project/test-infra/blob/master/prow/images/buildpack-golang/README.md) document.
 
-## Development
+## Installation 
 
-Before each commit, use the `before-commit.sh` script, which tests your changes.
+To run Helm-Broker you need a Kubernetes cluster with Tiller and Service-Catalog installed. To setup Helm-Broker in Minikube with all necessary dependencies follow these steps:
+
+Run the Minikube:
+```bash
+minikube start
+``` 
+
+If you don't have helm installed already, download [the Helm CLI](https://github.com/kubernetes/helm#install).
+Install Tiller into your cluster:
+```bash
+helm init
+```
+
+Install Service-Catalog as a Helm chart:
+```bash
+helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+helm install svc-cat/catalog --name catalog --namespace catalog
+```
+
+Install Helm-Broker chart:
+```bash
+helm install charts/helm-broker --name helm-broker --namespace helm-broker
+```
+## Usage
+
+This section explains how to use the Helm-Broker.
+
+### Provisioning services provided by the Helm-Broker
+
+If you have installed Helm-Broker with Service-Catalog, you can add an Addon repository and provision Redis instance.
 
-### Run a local version
+Add Addon repository configuration:
+```bash
+kubectl apply -f contrib/sample-addons.yaml
+```
+
+After the Helm-Broker processing the addons configuration you can see ClusterServiceClasses:
+
+```bash
+kubectl get clusterserviceclass
+```
+
+Provision Redis instance:
+```bash
+kubectl apply -f contrib/redis-instance.yaml
+```
+
+Now you can check the status of the Redis instance:
+```bash
+kubectl get serviceinstance
+```
```

Comment:
Add yaml to create service binding. Explain what is expected content of the secret.